app/views/checkouts/_checkouts_table.html.erb

```diff
@@ -1,6 +1,29 @@
 <%= form_with url: renewals_batch_update_path, method: :patch, id: 'checkouts' do %>
     <table id="checkouts-table" class="table table-responsive-sm" role="table" aria-live="polite">
-      <caption><h2>Checkouts</h2></caption>
+      <caption>
+        <span class="dropdown float-right">
+          <style>
+            .dropdown button,
+            .dropdown-menu {
+              font-family: "Open Sans", Arial, serif;
+            }
+          </style>
+          <button
+            class="btn btn-primary deflatten dropdown-toggle mb-2"
+            type="button"
+            id="export-checkouts-btn"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false">
+            Export All Checkouts
+          </button>
+          <div class="dropdown-menu" aria-labelledby="export-checkout-btn">
+            <p><%= link_to 'RIS File', "#{Settings.catalog.bulk_ris_url}#{(checkouts.map { |c| c.catkey.to_s }).join(',')}" %></p>
+            <%= link_to 'Email', export_checkouts_email_path(checkouts: checkouts.map { |c| { catkey: c.catkey, title: c.title, author: c.author, call_number: c.call_number } }) %>
+          </div>
```

Comment:
To format this dropdown in the bootstrap way i think we want it to look like this:

```
<div class="dropdown-menu" aria-labelledby="export-checkout-btn">
      <%= link_to 'RIS File', "#{Settings.catalog.bulk_ris_url}#{(checkouts.map { |c| c.catkey.to_s }).join(',')}", class: 'dropdown-item' %>
      <%= link_to 'Email', export_checkouts_email_path(checkouts: checkouts.map { |c| { catkey: c.catkey, title: c.title, author: c.author, call_number: c.call_number } }), class: 'dropdown-item' %>
</div>
```

Reference: https://getbootstrap.com/docs/4.0/components/dropdowns/